.github/workflows/docs.yml

```diff
@@ -37,6 +37,13 @@ jobs:
         shell: bash
         run: |
           tox -edocs
+      - name: Upload docs artifact
+        if: always()
+        uses: actions/upload-artifact@v3
```

Comment:
Does this allow for the docs to be built in parallel, or does it serve some other purpose?

Reply:
*sigh* I should've read PR description. Disregard :)

Reply:
No. All this does is that after the docs are built, we upload them to GitHub CI so that users can download the docs and see how they look on their local machine.

You already have parallel docs enabled with the `-j auto`:

https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/blob/c5207b191b601223e8e4e5eb56b2200c256101e7/tox.ini#L63